lib/views/help/privacy.html.erb

```diff
@@ -279,9 +279,11 @@
   </p>
   <p>
     We would also like to share non-personal data from WhatDoTheyKnow with
-    external researchers and are currently working on a policy which will allow
-    us to take precautions against the release of sensitive personal information
-    in cases where such data has been released in error. These precautions
+    external researchers where appropriate, in order to assist them with 
+    their work. When we do this we follow our Research Data Release Policy
```

Comment:
Does this have a public URL?

Reply:
It's linked after this paragraph, but not in the new text

  <p>
    Our Research Data Release policy can be viewed
    <a href="https://docs.google.com/document/d/1ytnM2U3fjMgri49YizFDt_9A-2uQmcF4qnGgFDMJpk0/edit#">here</a>.
  </p>

Reply:
Oh durr. Thanks 👍